js/how-does-javascript-work/how-does-javascript-work.md

```diff
@@ -0,0 +1,149 @@
+Hello, 👋 this is Shivam Verma, this is my first blog and in this, I will help you to understand the working of JavaScript. Let's Begin!!
+
+#### Ever wondered how JavaScript works under the hood?     
+
+Well by the end of this blog you will have a clear understanding of that by knowing two important concepts that make it possible
+1. **Execution Context**
+1. **Call Stack**          
+
+### 1. Execution Context
```

Comment:
Now, this is a sub sub heading under the above section "Ever wondered how JavaScript works under the hood?"

So it's okay to use `###` here

Always go down one level at a time

`###` will come after `##`

Using `####` just after `##` is not correct because we missed one level